pimd/pim_iface.c

```diff
@@ -1504,6 +1504,29 @@ void pim_if_create_pimreg(struct pim_instance *pim)
 	}
 }
 
+struct prefix* pim_if_find_connected_source(struct interface *ifp, struct in_addr src)
```

Comment:
This is just pim_if_connected_to_source very slightly different.  Why not see if we can abstract this so that we have 1 of these functions?